quantuminspire/sdk/qiskit/backend.py

```diff
@@ -211,9 +211,9 @@ def _generate_cqasm(
         number_of_qubits = experiment.header.n_qubits
         instructions = experiment.instructions
         with io.StringIO() as stream:
-            stream.write("version 1.0\n")
```

Comment:
During the refinement we discussed that we need a separate backend like QuantumInspireBackendV3 or just a single backend that has "cqasm_version" as an optional argument. Because there will be some amount of time that we need to support both V3 van V1 at the same time.

Reply:
Since we either produce v1 or v3, I added a boolean optional parameter to the backend constructor which by default will produce v1.